_data/download-descriptions.yml

```diff
@@ -257,23 +257,23 @@ source.zip:
 win.zip:
   description: Headers/libraries to compile with MSVC for Windows
 windows-arm64.exe:
-  description: Windows 10 (ARM64) (installer)
+  description: Windows 10 / 11 (ARM64) (installer)
 windows-arm64.pdb.xz:
   description: Debug Symbols for Windows (ARM64) (xz/lzma archive)
 windows-arm64.zip:
-  description: Windows 10 (ARM64) (zip archive)
+  description: Windows 10 / 11 (ARM64) (zip archive)
```

Comment:
@orudge : can you confirm Windows works on Win11 for ARM too?

(I can confirm the rest works on Win11 :P)

Reply:
@TrueBrain It did when I tested it some months ago :)